common/src/main/scala/org/apache/celeborn/common/util/Utils.scala

```diff
@@ -1092,4 +1093,8 @@ object Utils extends Logging {
   }
 
   def getProcessId: String = ManagementFactory.getRuntimeMXBean.getName.split("@")(0)
+
+  private val dateFmt: FastDateFormat =
```

Comment:
Could you also remove `dateFmt` of `HttpService` and replace the invoker of `dateFmt` with `Utils#formatTimestamp`?

Reply:
resolved